<ui version="4.0" >
 <class>Ui::ABRepeatDialog</class>
 <widget class="QDialog" name="Ui::ABRepeatDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>반복 구간</string>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>A:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="a_time_edit" >
        <property name="displayFormat" >
         <string>h:mm:ss.zzz</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>B:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>0</number>
        </property>
        <item>
         <widget class="QTimeEdit" name="b_time_edit" >
          <property name="displayFormat" >
           <string>h:mm:ss.zzz</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>자동입력</string>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label_5" >
          <property name="text" >
           <string>A 입력</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QPushButton" name="a_time_button" >
          <property name="text" >
           <string>현재 시간(&amp;A)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="QPushButton" name="a_subtitle_button" >
          <property name="text" >
           <string>현재 자막 시작(&amp;S)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_6" >
          <property name="text" >
           <string>B 입력</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QPushButton" name="b_time_button" >
          <property name="text" >
           <string>현재 시간(&amp;B)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" >
         <widget class="QPushButton" name="b_subtitle_button" >
          <property name="text" >
           <string>현재 자막 종료(&amp;E)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>반복횟수: </string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>0</number>
       </property>
       <item>
        <widget class="QSpinBox" name="times_spin" >
         <property name="specialValueText" >
          <string>무한반복</string>
         </property>
         <property name="accelerated" >
          <bool>true</bool>
         </property>
         <property name="suffix" >
          <string> 번</string>
         </property>
         <property name="maximum" >
          <number>999</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>앞뒤로</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="add_time_spin" >
       <property name="accelerated" >
        <bool>true</bool>
       </property>
       <property name="suffix" >
        <string> 초</string>
       </property>
       <property name="decimals" >
        <number>1</number>
       </property>
       <property name="maximum" >
        <double>9.000000000000000</double>
       </property>
       <property name="singleStep" >
        <double>0.100000000000000</double>
       </property>
       <property name="value" >
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7" >
       <property name="text" >
        <string>추가</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="start_button" >
       <property name="text" >
        <string>반복시작</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quit_button" >
       <property name="text" >
        <string>종료</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="close_button" >
       <property name="text" >
        <string>닫기</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>close_button</sender>
   <signal>clicked()</signal>
   <receiver>Ui::ABRepeatDialog</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>280</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel" >
     <x>295</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
